android/src/com/mapswithme/maps/maplayer/traffic/widget/TrafficButton.java

```diff
@@ -36,7 +39,7 @@ private static AnimationDrawable getLoadingAnim(@NonNull ImageButton trafficBtn)
     Context context = trafficBtn.getContext();
     Resources res = context.getResources();
     final int animResId = ThemeUtils.getResource(context, R.attr.trafficLoadingAnimation);
-    return (AnimationDrawable) res.getDrawable(animResId, context.getTheme());
+    return (AnimationDrawable) Objects.requireNonNull(ResourcesCompat.getDrawable(res, animResId, context.getTheme()));
```

Comment:
nit: Do we really should write this ugly expression Objects.requireNonNull(...) even if the function is already marked as NonNull ?

Reply:
ResourcesCompat.getDrawable() is declared `@Nullable`.